tests/phpunit/test-class-wc-stripe-notes.php

```diff
@@ -102,3 +115,7 @@ public function test_create_upe_availability_note_does_not_create_note_when_upe_
 		$this->assertSame( 0, count( $admin_note_store->get_notes_with_name( WC_Stripe_UPE_Availability_Note::NOTE_NAME ) ) );
 	}
 }
+
+//function woocommerce_gateway_stripe() {
+//	return WC_Stripe_Inbox_Notes_Test::$global_woocommerce_gateway_stripe;
+//}
```

Comment:
Do we still need these? :D

Reply:
haha woops - I was trying different testing approaches :D Thanks! fixed in aa758710